src/inc/til/format.h

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <stdint.h>
+#include <string>
+
+#ifdef UNIT_TESTING
+#endif
+
+namespace til // Terminal Implementation Library. Also: "Today I Learned"
+{
+    namespace format
+    {
+        inline char* format_uint8(char* ptr, uint8_t number)
+        {
+            if (number >= 100)
+            {
+                *ptr++ = static_cast<char>((number / 100) + '0');
+            }
+            if (number >= 10)
+            {
+                *ptr++ = static_cast<char>(((number / 10) % 10) + '0');
+            }
+
+            *ptr++ = static_cast<char>((number % 10) + '0');
+            return ptr;
+        }
+
+        inline char* format(char* pDst, const char* pFormat, ...)

```

Comment:
This is very drafty naming. Ideas are more than welcomed.

Reply:
Is there a way I can restrict the `va_args` to only support `uint8_t`? That way I can make sure this method is safe to use.